src/components/sanity/readmore/ReadMore.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { TypedObject } from "@portabletext/types";
+import { PortableText } from "@portabletext/react";
+import { PortableTextComponentProps } from "@portabletext/react/src/types";
+import { ReadMore as DSReadmore } from "@navikt/ds-react";
```

Comment:
Jeg vil gjerne ha tilbakemelding på dette. I denne komponenten porter vi innholdet fra Sanity videre til designsystemet sin egen `ReadMore` komponent. Er `DSReadmore` en OK måte å løse opp i navngivingsproblemene her?

Reply:
Ønsker du at `DSReadmore` skal være alle steder? Lenger opp i reviewen, så har du brukt `readMore` Hva står DS for?

Reply:
Jeg synes dette er en fin måte å gjøre det på :) Kan evt bytte navn til NAVReadmore hvis det er mer beskrivende, men jeg har ingen sterke preferanser der.

Reply:
Jeg synes det er en fin løsning det! Som Sindri sa, NAVReadmore funker og. Du får se hva du fortrekker!

Reply:
Supert! Da endrer jeg til `NAVReadmore` 😊